searx/settings.yml

```diff
@@ -2037,6 +2037,15 @@ engines:
     categories: videos
     disabled: true
 
+  - name: livespace
+    engine: livespace
+    shortcut: ls
+    base_url: https://live.space/
+    paging: true
```

Comment:
That line can be removed, since you already set `paging: true` in `livespace.py`